<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>126</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="14">
    <widget class="QLabel" name="delayLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>td: 0 мкс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="15">
    <widget class="QPushButton" name="startStopButton">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #ee0000;
	border: 1px solid #ee0000;
	border-radius: 4px;
	color: #fff;
}

/*QPushButton:checked {
	background-color: #00ee00;
	border: 1px solid #00ee00;
}*/</string>
     </property>
     <property name="text">
      <string>Остановлен</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="Line" name="line_11">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="masterSlaveButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #00ee00;
	border: 1px solid #00ee00;
	border-radius: 4px;
	color: #fff;
}

/*QPushButton:checked {
	background-color: #ee0000;
	border: 1px solid #ee0000;
}*/</string>
     </property>
     <property name="text">
      <string>slave</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="linkLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	width: 20px;
	height: 20px;
    border-radius: 10px;
    border: 1px solid rgb(93, 231, 33);
	background-color: rgb(93, 231, 33);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="devNameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>#0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="durationLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0 мкс</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="13">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="17">
    <widget class="QPushButton" name="settingsButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>V</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="frequencyLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0.0 Гц</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLabel" name="voltageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0.0 В</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="currentLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0.0 A</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="18">
    <widget class="QGroupBox" name="controlBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>78</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="6">
       <widget class="QLabel" name="durationUnitLabel_2">
        <property name="text">
         <string>мкс</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="currentSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="devNameEdit"/>
      </item>
      <item row="0" column="10" rowspan="3">
       <widget class="Line" name="line_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="14" rowspan="3">
       <widget class="Line" name="line_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QDoubleSpinBox" name="voltageSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QDoubleSpinBox" name="durationSpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="15">
       <widget class="QDoubleSpinBox" name="delaySpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QDoubleSpinBox" name="frequencySpinBox">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="16">
       <widget class="QLabel" name="delayUnitLabel_2">
        <property name="text">
         <string>мкс</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="currentUnitlabel">
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="maxDurationLabel">
        <property name="text">
         <string>max: 0.00 мкс</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="maxCurrentLabel">
        <property name="text">
         <string>max: 0.00 A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="13">
       <widget class="QLabel" name="voltageUnitLabel_2">
        <property name="text">
         <string>В</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="3">
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="frequencyUnitLabel_2">
        <property name="text">
         <string>Гц</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7" rowspan="3">
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="maxFrequencyLabel">
        <property name="text">
         <string>max: 0.00 Гц</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="maxVoltageLabel">
        <property name="text">
         <string>max: 0.00 В</string>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QLabel" name="maxDelayLabel">
        <property name="text">
         <string>max: 0.00 мкс</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="minCurrentLabel">
        <property name="text">
         <string>min: 0.00 A</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QLabel" name="minDurationLabel">
        <property name="text">
         <string>min: 0.00 мкс</string>
        </property>
       </widget>
      </item>
      <item row="2" column="8">
       <widget class="QLabel" name="minFrequencyLabel">
        <property name="text">
         <string>min: 0.00 Гц</string>
        </property>
       </widget>
      </item>
      <item row="2" column="12">
       <widget class="QLabel" name="minVoltageLabel">
        <property name="text">
         <string>min: 0.00 В</string>
        </property>
       </widget>
      </item>
      <item row="2" column="15">
       <widget class="QLabel" name="minDelayLabel">
        <property name="text">
         <string>min: 0.00 мкс</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="powerButton">
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>23</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: #ee0000;
	border: 1px solid #ee0000;
	border-radius: 4px;
	color: #fff;
}

/*QPushButton:checked {
	background-color: #00ee00;
	border: 1px solid #00ee00;
}*/</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
